save-cloud-charts/save-cloud/README.md

```diff
@@ -0,0 +1,16 @@
+# Helm chart for save-cloud
```

Comment:
pls add links for a local installation of Kuber

Reply:
Added